src/status_im/contexts/wallet/common/token_value/view.cljs

```diff
@@ -71,47 +73,36 @@
 
 (defn token-value-drawer
   [token watch-only? entry-point]
-  (let [token-symbol           (:token token)
-        token-data             (first (rf/sub [:wallet/current-viewing-account-tokens-filtered
-                                               {:query token-symbol}]))
-        fiat-unformatted-value (get-in token [:values :fiat-unformatted-value])
-        has-balance?           (money/greater-than fiat-unformatted-value (money/bignumber "0"))
-        selected-account?      (rf/sub [:wallet/current-viewing-account-address])
-        token-owners           (rf/sub [:wallet/operable-addresses-with-token-symbol token-symbol])
-        send-params            (if selected-account?
-                                 {:token       token-data
-                                  :stack-id    :screen/wallet.accounts
-                                  :disabled?   (not has-balance?)
-                                  :start-flow? true
-                                  :owners      token-owners}
-                                 {:token-symbol token-symbol
-                                  :stack-id     :wallet-stack
-                                  :start-flow?  true
-                                  :owners       token-owners})
-        bridge-params          (if selected-account?
-                                 {:token            token-data
-                                  :bridge-disabled? (or (not has-balance?)
-                                                        (send-utils/bridge-disabled? token-symbol))
-                                  :stack-id         :screen/wallet.accounts
-                                  :start-flow?      true
-                                  :owners           token-owners}
-                                 {:token-symbol     token-symbol
-                                  :bridge-disabled? (send-utils/bridge-disabled? token-symbol)
-                                  :stack-id         :wallet-stack
-                                  :start-flow?      true
-                                  :owners           token-owners})]
```

Comment:
Removing these changes may affect bridge and send flows navigation. Are you sure we want to remove them?

Reply:
They are not getting removed here, it's simply a small restructuring because some of the code is redundant.